lib/KSelect/KeenUiSelect.vue

```diff
@@ -666,7 +666,7 @@
         }
 
         this.$emit('select', option, {
-          selected: this.multiple ? shouldSelect : true,
+          selected: this.isOptionSelected(option),
```

Comment:
This change means that the `selected` property is always false for a newly selected item (it's possible to test this using the vue dev tools to inspect the select event on the documentation).

I think `isOptionSelected` might be reporting the state prior to the select, so I think it might be safe to just report the negation:
```suggestion
          selected: !this.isOptionSelected(option),
```

Reply:
Oooh, yes, I was seeing the `selected` property as "previously selected", "This item that you have selected was selected before?", but now I see that should mean "Has the new item been selected?" and say true if it has not been selected before and should be selected now , and false otherwise

Reply:
Thanks @rtibbles! Changes made :)